Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -2370,7 +2375,9 @@ public KeyControl this[Key key]
             {
                 var index = (int)key - 1;
                 if (index < 0 || index >= m_Keys.Length)
-                    throw new ArgumentOutOfRangeException(nameof(key));
+                {
+                    throw new ArgumentOutOfRangeException($"{nameof(key)}: {key}");
+                }
                 return m_Keys[index];
```

Comment:
I am leaving towards it would be better to return null for index == (int)Key.IMESelected - 1?